lib/matplotlib/cbook/__init__.py

```diff
@@ -543,6 +543,7 @@ def file_requires_unicode(x):
         return False
 
 
+@deprecated('2.1')
 def is_scalar(obj):
     """return true if *obj* is not string like and is not iterable"""
     return not is_string_like(obj) and not iterable(obj)
```

Comment:
'cause that confuses me 'cause aren't scalers numbers? What definition of scalers are we using then?